### ISSUE TITLE
How to evaluate the results on test set with ground truths

### ISSUE THREAD

**Issue by amardeepjaiman** [Oslandia/mapillary_classification#137]
Hi,

can someone guide me how can we evaluate the model on test set using the predictions and ground truths.

Thanks,
Amardeep

**Comment by delhomer** [comment]
Hi @amardeepjaiman,

Actually in this project we do not evaluate the model regarding the test set (its done only with training and validation sets during training process). However if you get a ground-truth version of an image and a semantic segmentation prediction (let say, as two `.png` files), you can substitute pixel triplets by corresponding label IDs (see some functions in `generator.py` that deal with this process) and compute metrics of your choice (*ex* accuracy, IoU) by comparing both structures in a `a posteriori` logic.

Contributions are welcome ! :)

**Comment by delhomer** [comment]
As for #136, there is no news here, I close this issue.
